devicedb/rpm/files/devicedb.service

```diff
@@ -1,7 +1,11 @@
 [Unit]
 Description=DeviceDB: Remote database manager for Maestro
-After=maestro.service edge-proxy.service
-Wants=maestro.service edge-proxy.service
+PartOf=pelion.service
+After=pelion.service
+# maestro generates the devicedb config file from a template
+After=maestro.service
```

Comment:
Can deviceDB run without maestro started?  If not, then [we need something along with `After` here(`Requires`?)](https://serverfault.com/questions/812584/in-systemd-whats-the-difference-between-after-and-requires)

Reply:
It can in the normal case, but not firstboot.  The first time the system boots, maestro generates devicedb's config file from a template.  The config file is written to persistent storage, so devicedb shouldn't require maestro from then on.

Reply:
We'll need to add something here then.  Maybe `Wants` or `Requires`?